plugins/catalog-backend/src/catalog/CatalogLogic.ts

```diff
@@ -15,13 +15,14 @@
  */
 
 import { Logger } from 'winston';
-import { LocationReader } from '../ingestion/types';
+import { parserOutputOnlyContainsErrors } from '../descriptors';
+import { readLocation } from '../ingestion';
 import { Catalog } from './types';
 
 export class CatalogLogic {
   public static startRefreshLoop(
     catalog: Catalog,
-    reader: LocationReader,
+    reader: typeof readLocation,
```

Comment:
Think it would be nicer to keep the named function type, e.g. `LocationReaderFunc`